configuration/cardano/mainnet-config.json

```diff
@@ -11,6 +11,7 @@
   "RequiresNetworkMagic": "RequiresNoMagic",
   "ShelleyGenesisFile": "mainnet-shelley-genesis.json",
   "ShelleyGenesisHash": "1a3be38bcbb7911969283716ad7aa550250226b76a61fc51cc9a9a35d9276d81",
+  "ShelleyHardForkNotBeforeEpoch": 207,
```

Comment:
Edsko confirmed to me that we should actually use 208 here, not 207.

Reply:
Updated.